client-src/overlay.js

```diff
@@ -138,7 +137,7 @@ function show(messages, type) {
 
       // Make it look similar to our terminal.
       const errorMessage = message.message || messages[0];
-      const text = ansiHTML(encode(errorMessage));
```

Comment:
It is not safe, here can be XSS

Reply:
Maybe we need [decode(text, {level: 'html5'})](https://github.com/mdevils/html-entities#decodetext-options) afterwards

Reply:
hm, maybe, just try it